agent/engine/task_manager.go

```diff
@@ -278,6 +278,12 @@ func (mtask *managedTask) handleContainerChange(containerChange dockerContainerC
 		}
 	}
 
+	seelog.Debugf("Sending container change event to tcs, container: %v, status: %v", event.DockerId, event.Status)
```

Comment:
Can you change this message?  It's not emitting it only to `tcs` but to any handler subscribed to the event stream.  Perhaps "Sending container change event to event stream, container: %v, status: %v"?